bin/smart_proxy_dynflow_core

```diff
@@ -3,5 +3,22 @@ require 'dynflow'
 require 'rack'
 require 'smart_proxy_dynflow_core/launcher'
 require 'yaml'
+require 'optparse'
```

Comment:
it seems we have clamp packaged in SCL and nonSCL form, maybe we could use that instead of optparse?